docs/source/contributing/documentation.rst

```diff
@@ -318,6 +318,24 @@ elements that are either a ``str`` or ``None``;
 ``(str, int)``; ``Tuple[:class:`int`, ...]`` for a tuple of variable
 length with only integers.
 
+.. note::
+   As a helper for tool for typesets, you can use ``typestring-parser`` which can 
+   be accessed by first downloading it via ``pip`` - ``pip install typestring-parser`` and
+   then using ``from typestring_parser import parse``.
+
+   Example::
+
+    from typestring_parser import parse
+    
+    >>> parse('int')
+    <class 'int'>
+    >>> parse('int or str')
+    typing.Union[int, str]
+    >>> parse('list of str or str')
+    typing.Union[typing.List[str], str]
+    >>> parse('list of (int, str)')
+    typing.List[typing.Tuple[int, str]]
```

Comment:
this doesn't render properly with syntax highlighting
![image](https://user-images.githubusercontent.com/49693820/116426017-62a15880-a860-11eb-90ec-1b1c62a6a5a8.png)

Reply:
No let's not have it as it just makes the CI fail.

Reply:
I would think there's some way to have a code block highlighted in the documentation, but not run - right? @naveen521kk 

If not, this is definitely not a breaking change.

Reply:
@Nikhil0504 I know how to do this *without breaking* things, but I want you to search or rather explore to find things out. 
You can easily add syntax highlighting, browse through though how it is done in other parts of manim itself or look at the reStructuredText Primer from sphinx or search the internet.

Reply:
Hey, I tried fixing the issue and couldn't find a solution for it. Is it possible for you to fix it?

Reply:
Take a look at how https://docs.manim.community/en/latest/tutorials/quickstart.html does it